region/wreckedship/main.json

```diff
@@ -1342,7 +1509,29 @@
               "yields": [ "f_ZebesAwake" ],
               "devNote":  "This is unlocked and yielded from killing the enemies, which are free to kill"
             }
-          ]
+          ],
+          "gModeImmobile": {
+            "requires": [
+              {"or": [
+                {"enemyDamage": {
+                  "enemy": "Kihunter (yellow)",
+                  "type": "contact",
+                  "hits": 1
+                }},
+                {"and": [
+                  {"enemyDamage": {
+                    "enemy": "Covern",
+                    "type": "contact",
+                    "hits": 1
+                  }},
+                  { "not": "f_DefeatedPhantoon" },
+                  "canRiskPermanentLossOfAccess"
+                ]}
+              ]}
+            ],
+            "note": "Use a Covern or Kihunter. A flying Kihunter takes about 25 seconds to get to the door.",
```

Comment:
I timed 12 seconds.

Reply:
Good catch. Looks like I can get it to range between 11 and 27 seconds. I updated the description, I mostly just didnt want people to think it was broken because the 2 ground Kihunters don't hit Samus.